testctrl/svc/orch/controller.go

```diff
@@ -82,6 +85,13 @@ func (c *Controller) Schedule(s *types.Session) error {
 	}
 
 	c.waitQueue.Enqueue(s)
+	if c.store != nil {
+		c.store.StoreEvent(s.Name, &types.Event{
+			SubjectName: s.Name,
+			Kind:        types.QueueEvent,
+			Time:        time.Now(),
+		})
+	}
```

Comment:
I did not use the Event constructor, because not all events have the same fields. Some have logs, some do not. It also did not look very aesthetically appealing. I think it is better to use events as literals.